Gianluca's fix for 23817

lgarren opened the pull request.

FNALbuild commented:
The code-checks are being triggered in jenkins.

FNALbuild commented:
-code-checks 
 Unable to merge PR 1 into develop branch of LArSoft/lardata. Please resolve merge conflicts. 
 for details see 
 https://buildmaster.fnal.gov/buildmaster/job/GithubIntegration/job/run-pr-code-checks//182/console

FNALbuild commented:
A new Pull Request was created by  @gartung (Patrick Gartung) for develop.

It involves the following packages:

lardata


@gartung, @FNALbuild can you please review it and eventually sign? Thanks.
@gartung this is something you requested to watch as well.

cms-bot commands are listed <a href="http://cms-sw.github.io/cms-bot-cmssw-cmds.html">here</a>

PetrilloAtWork committed:
Rewritten `lar::util::TrackPitchInView()`.

This is a candidate solution for issue #23817.
This also should make the function geometry-independent.

PetrilloAtWork committed:
Alignment and style update.

PetrilloAtWork committed:
Using default parameter type for RealComparisons.

PetrilloAtWork committed:
Added disabled code for the future come of `InterWireProjectedDistance()`.

PetrilloAtWork committed:
Merge remote-tracking branch 'origin/develop' into feature/gp_Issue23817

PetrilloAtWork committed:
Updated the disabled code to react to the new behaviour of `InterWireProjectedDistance()`.

lgarren committed:
whitespace fix
